apps/teams-test-app/src/App.tsx

```diff
@@ -135,6 +136,7 @@ const App = (): ReactElement => {
       <PagesBackStackAPIs />
       <PagesConfigAPIs />
       <PagesTabsAPIs />
+      <PagesSelfAPIs />
```

Comment:
It looks like this list is alphabetized, so we should move PagesSelfAPIs above PagesTabsAPIs